userLogin.js

```diff
@@ -0,0 +1,33 @@
+function buttonClick() {
+    var e = document.getElementById("loginButton");
```

Comment:
This e could be named better to represent what it is.

ex: 

var loginButtonElement
var button
var element
etc.